watertap/property_models/tests/test_ion_DSPMDE_prop_pack.py

```diff
@@ -0,0 +1,450 @@
+                        PhysicalParameterBlock,
+                        StateBlockData,
+                        StateBlock,
+                        MaterialBalanceType,
+                        EnergyBalanceType,
+                        )
+from idaes.core.util.scaling import calculate_scaling_factors, set_scaling_factor, get_scaling_factor
+
+from pyomo.util.check_units import assert_units_consistent
+from watertap.property_models.ion_DSPMDE_prop_pack import DSPMDEParameterBlock, DSPMDEStateBlock, ActivityCoefficientModel
+from idaes.generic_models.properties.tests.test_harness import \
+    PropertyTestHarness as PropertyTestHarness_idaes
+from watertap.property_models.tests.property_test_harness import \
+    (PropertyTestHarness, PropertyRegressionTest, PropertyCalculateStateTest)
+from watertap.core.util.initialization import check_dof
+from idaes.core.util.model_statistics import *
+from idaes.core.util.scaling import unscaled_variables_generator, unscaled_constraints_generator, badly_scaled_var_generator
+from watertap.property_models.tests.property_test_harness import PropertyAttributeError
+from watertap.property_models.tests.property_test_harness import PropertyTestHarness
+from idaes.core.util import get_solver
+
+solver = get_solver()
+# -----------------------------------------------------------------------------
+
+@pytest.fixture(scope="module")
+def model():
+    m = ConcreteModel()
+
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+    m.fs.properties = DSPMDEParameterBlock(default={
+        "solute_list": ["A", "B", "C", "D"],
+        "diffusivity_data": {("Liq", "A"): 1e-9,
+                             ("Liq", "B"): 1e-10,
+                             ("Liq", "C"): 1e-7,
+                             ("Liq", "D"): 1e-11},
+        "mw_data": {"H2O": 18e-3,
+                    "A": 10e-3,
+                    "B": 25e-3,
+                    "C": 100e-3,
+                    "D": 25e-3},
+        "stokes_radius_data": {"A": 1e-9,
+                               "B": 1e-9,
+                               "C": 1e-9,
+                               "D": 1e-10},
+        "charge": {"A": 1,
+                   "B": -2,
+                   "C": 2,
+                   "D": -1}
+    })
+
+    return m
+
+@pytest.mark.unit
+def test_parameter_block(model):
+    assert isinstance(model.fs.properties.component_list, Set)
+    for j in model.fs.properties.component_list:
+        assert j in ["H2O", "A", "B", "C", "D"]
+    assert isinstance(model.fs.properties.solvent_set, Set)
+    for j in model.fs.properties.solvent_set:
+        assert j in ["H2O"]
+    assert isinstance(model.fs.properties.solute_set, Set)
+    for j in model.fs.properties.solute_set:
+        assert j in ["A", "B", "C", "D"]
+
+    assert isinstance(model.fs.properties.phase_list, Set)
+    for j in model.fs.properties.phase_list:
+        assert j in ["Liq"]
+
+    assert model.fs.properties._state_block_class is DSPMDEStateBlock
```

Comment:
```suggestion
    assert model.fs.properties._state_block_class is PropStateBlock
```

Reply:
@andrewlee94 any objections

> My suggestions are based on your first TODO note and what Tim had mentioned before about standardizing the class names of property packages. It is probably better to go with the standardized names now (if that is what we are going to be doing), then have to make a bunch of changes in the code later.

@aladshaw3 - given @andrewlee94 's feedback and thinking about it more, it might be best to leave things as they were instead of trying to standardize (e.g.., do not shift to using `PropStateBlock` instead of `DSPMDEStateBlock`). If we decide that's best for now, I'd just delete the to do comment.

Reply:
Ok, that sounds fine with me.